src/test/java/org/cactoos/io/HeadInputTest.java

```diff
@@ -39,30 +39,30 @@
 
     @Test
     public void readsHeadOfLongerInput() throws Exception {
-        MatcherAssert.assertThat(
-            "HeadInput couldn't limit a number of read bytes",
+        new Assertion<>(
+            "must limit the number of read bytes",
             new TextOf(
                 new HeadInput(
                     new InputOf("readsHeadOfLongerInput"),
                     5
                 )
             ),
             new TextHasString("reads")
-        );
+        ).affirm();
     }
 
     @Test
     public void readsHeadOfShorterInput() throws Exception {
         final String input = "readsHeadOfShorterInput";
-        MatcherAssert.assertThat(
-            "HeadInput incorrectly limited a number of read bytes",
+        new Assertion<>(
+            "must limit a number of read bytes",
```

Comment:
@victornoel minor comment: check alignment with message on line 43. Also I think the semantics could be improved in both cases for these messages (they're a little generic on the expected result for my taste).